src/Symfony/Bridge/Doctrine/Form/ChoiceList/EntityChoiceList.php

```diff
@@ -99,7 +99,7 @@ class EntityChoiceList extends ObjectChoiceList
      *                                                     the choices are given as flat array.
      * @param PropertyAccessorInterface $propertyAccessor  The reflection graph for reading property paths.
      */
-    public function __construct(ObjectManager $manager, $class, $labelPath = null, EntityLoaderInterface $entityLoader = null, $entities = null, array $preferredEntities = array(), $groupPath = null, PropertyAccessorInterface $propertyAccessor = null)
+    public function __construct(ObjectManager $manager, $class, $labelPath = null, EntityLoaderInterface $entityLoader, $entities = null, array $preferredEntities = array(), $groupPath = null, PropertyAccessorInterface $propertyAccessor = null)
```

Comment:
So not to break BC, would it make sense to leave `$entityLoader` is nullable and construct one using the required `$manager` and `$class` arguments?

```
$this->entityLoader = ($entityLoader !== null)
    ? $entityLoader
    : $manager->getRepository($class)->createQueryBuilder('e');
```